wire-ios/WireUITests/PersonalUserTests.swift

```diff
@@ -62,5 +62,24 @@ final class PersonalUsersTests: WireUITestCase {
 
         try accountPage.logout()
             .enterPassword(user.password)
+
+        XCTAssertTrue(welcomePage.emailTextField.exists, "Account logout failed")
+    }
+
+    @MainActor
+    func test_Login_asExistingPersonalUser() async throws {
+        let user = try await userManager.createPersonalUser()
+
+        let welcomePage = try WelcomePage()
+            .enterEmailOrSSO(user.email)
+            .enterPassword(user.password)
+            .acceptFirstTimeAlert()
+            .acceptPopup()
+            .openSettings()
+            .openAccountSettings()
+            .logout()
+            .enterPassword(user.password)
+
+        XCTAssertTrue(welcomePage.emailTextField.exists, "Account logout failed")
```

Comment:
suggestion: Can we use welcome page's method to know if it loaded for this check?